src/lib/PostgrestQueryBuilder.ts

```diff
@@ -21,7 +21,7 @@ export default class PostgrestQueryBuilder<T> extends PostgrestBuilder<T> {
    * @param count  Count algorithm to use to count rows in a table.
    */
   select(
-    columns = '*',
+    columns: '*' | string | keyof T | Array<keyof T> = '*',
```

Comment:
This will show `'*'`, and the column names as suggestions, while still allowing a comma-separated string